tests/project/test_mbed_project.py

```diff
@@ -24,55 +30,56 @@ def mock_git():
 
 
 class TestInitialiseProject:
-    def test_fetches_mbed_os_when_create_only_is_false(self, mock_program):
+    def test_fetches_mbed_os_when_create_only_is_false(self, mock_libs, mock_program):
         path = pathlib.Path()
         initialise_project(path, create_only=False)
 
         mock_program.from_new.assert_called_once_with(path)
-        mock_program.from_new.return_value.resolve_libraries.assert_called_once()
+        mock_libs().fetch.assert_called_once()
 
-    def test_skips_mbed_os_when_create_only_is_true(self, mock_program):
+    def test_skips_mbed_os_when_create_only_is_true(self, mock_program, mock_libs):
```

Comment:
nitpick: Could we keep the order of mock_program and mock_libs consistent between tests (all in this file with similar shape)?

Reply:
Fixed.